init.go

```diff
@@ -166,6 +167,7 @@ func init() {
 	inventoryClient.Default = inventoryClient.New(transport, nil)
 	managementClient.Default = managementClient.New(transport, nil)
 	serverClient.Default = serverClient.New(transport, nil)
+	amclient.Default.SetTransport(httptransport.New("127.0.0.1", "/alertmanager/api/v2", []string{"http"}))
```

Comment:
Here we ignore serverURLF value, and use hard-coded address, scheme – and no username and password. That's why it does not work.

Reply:
thanks , added auth